homeassistant/components/venstar/binary_sensor.py

```diff
@@ -14,12 +14,12 @@ async def async_setup_entry(hass, config_entry, async_add_entities) -> None:
 
     if coordinator.client.alerts is None:
         return
-    sensors = [
-        VenstarBinarySensor(coordinator, config_entry, alert["name"])
-        for alert in coordinator.client.alerts
-    ]
-
-    async_add_entities(sensors, True)
+    async_add_entities(
+        [
+            VenstarBinarySensor(coordinator, config_entry, alert["name"])
+            for alert in coordinator.client.alerts
+        ]
+    )
```

Comment:
We can pass a generator expression and if we don't pass other arguments, we don't need extra parentheses.
```suggestion
    async_add_entities(
        VenstarBinarySensor(coordinator, config_entry, alert["name"])
        for alert in coordinator.client.alerts
    )
```